### DISCUSSION TITLE
Does the flatpak version of bottles include gamemode package?

### DISCUSSION THREAD

**Discussion by Raj2032** [bottlesdevs/Bottles#3635] [Q&A]
I installed the flatpak version of bottles on my arch-based linux system.

With this "Feral GameMode" setting:

![image](https://github.com/user-attachments/assets/60d73132-2967-457c-ac78-e560a4416a31)

do I need to actually install [gamemode](https://github.com/FeralInteractive/gamemode) from my arch based system and configure it to run or its not needed as installing bottles through flatpak already includes gamemode?

**Comment by EmoonX** [comment]
We had a quite recent discussion about this on Discord. The overall conclusion is that, while Gamemode libraries are in fact bundled with Bottles and it _does_ work in some form, you need it installed on your host system to take full advantage of system-wide optimizations (like CPU governor and process niceness).

So just install `gamemode` through your package manager and Bottles should be able to communicate with it for maximum performance.

**Comment by Raj2032** [comment]
Hi, thanks for your comment, if bottles is sandboxed under flatpak, then how does it interact with the gamermode one that is installed on host machine?

**Reply by EmoonX** [reply]
It seems do so by a so-called _XDG desktop portal_: [https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.GameMode.html](https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.GameMode.html)

**Reply by Raj2032** [reply]
@EmoonX Makes sense!
